qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -210,20 +213,20 @@ def get_solver(  # type: ignore[override]
         ansatz.num_spin_orbitals = num_spin_orbitals
         ansatz.initial_state = initial_state
 
-        if isinstance(self.initial_point, InitialPoint):
-            self.initial_point.grouped_property = driver_result
+        if isinstance(self.initial_point, HFInitialPoint):
```

Comment:
```suggestion
        if isinstance(self.initial_point, InitialPoint):

Reply:
Incorporated this change as part of [8c5b10f](https://github.com/Qiskit/qiskit-nature/pull/632/commits/8c5b10fc437a19c4020ce2b57d34afd5966beb3c).